website/docs/developer/development-guide/code/linting.md

```diff
@@ -0,0 +1,13 @@
+---
+id: linting
+title: Running PyLint Against Your Code
+description: Instructions on running PyLint.
+---
+
+## Running PyLint
+1. Navigate to the task's base folder.
+1. Activate the virtual environment.
+1. Run pylint and fix any discovered issues if you can.
```

Comment:
May want to add some comments about creating/maintaining a .pylintrc file to handle false positives. Possibly add some reference links for pylint.

Reply:
Task Created